mmv1/third_party/terraform/services/compute/compute_instance_helpers.go.erb

```diff
@@ -482,6 +483,7 @@ func expandNetworkInterfaces(d tpgresource.TerraformResourceData, config *transp
 			StackType:         data["stack_type"].(string),
 			QueueCount:        int64(data["queue_count"].(int)),
 			Ipv6AccessConfigs: expandIpv6AccessConfigs(data["ipv6_access_config"].([]interface{})),
+			Ipv6Address:       data["ipv6_address"].(string),
```

Comment:
Looks like the error message pointed here:
panic: interface conversion: interface {} is nil, not string